### PR TITLE
Creating iraqiggc.txt which is Iraqi Gifted Student School emails domain

### PR DESCRIPTION
**Pull request by AnasAlQazaz**

### CODE REVIEW

**Comment by philipto** [comment]
@AnasAlQazaz Please provide us with:
- the school official website URL
- the school street address, including city and country
- an URL of a page on the official website where the school offers an IT-related long-term (>1 year) course
- a proof, which shows that the school recognizes the domain you are submitting as an official email domain for the students.

**Comment by philipto** [comment]
@AnasAlQazaz You might miss my previous comment. Please react.

**Comment by philipto** [comment]
@AnasAlQazaz I am sorry, this request remains unanswered for a while. I have to close it without merging. Please reopen and comment if you have an update regarding the aforementioned issue. Thank you for your understanding!

**Comment by AnasAlQazaz** [comment]
Hi Philip
I'm from Iraq and I've mentioned my problem on my request that was sent to
your support team ( #3565064
<https://sales.jetbrains.com/hc/requests/3565064> is the request number)

1-https://www.iraqiggc.edu.iq/
2-Gifted Students Schools are located in 5 governorates in Iraq and we all
have the same emails domain and all schools do computer science projects
and research paper
but for Ninaveh building location, here is the location on google map
https://goo.gl/maps/z1Y2ZPYYTXmSFJKH7

3-I can't really provide the third requirement because as I mentioned in my
request, we are a school that teaches regular iraqi highschool curriculum
but we also do research papers, computer science and robotics project, and
this year we are in need to use jetBrains products
in other words, my school doesn't have a courses and etc that are
available for everyone, although it's a non paid school.

4-I can't provide the fourth requirement, because as mentioned I live in
Iraq which is an underdeveloped country and it's hard to get documents like
this
but here is the facebook link for GSS of nineveh which I'm currently
studying at https://www.facebook.com/GSS.MOSUL/
